### DISCUSSION TITLE
Can I limit agent memory

### DISCUSSION THREAD

**Discussion by Tean** [wazuh/wazuh#15686] [Q&A] [answered]
Hello, is there any way to limit agent memory when using FIM? Because when I try add some agents to the test group which I config watch more other directories, they will use all memory & cache, but I can't release other process or memory, so I just want to know is there have any solution? the syscheck db is default config, thanks.

**Comment by mdiego92** [comment, marked as answer]
Hello @Tean,

When using FIM in your agents there are a few parameters in the `internal_options.conf` file that you can modify:

![image](https://user-images.githubusercontent.com/62477966/207881042-abc799ad-18d8-4a23-8904-db95139ec79f.png)

You can also change some parameters from the `ossec.conf` file to set a more granular scan ignoring specific file extensions or even directories that you don't consider useful.

**Articles for reference:**
[Syscheck: internal_options.conf](https://documentation.wazuh.com/current/user-manual/reference/internal-options.html#syscheck)
[Syscheck: ignore](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/syscheck.html#ignore)

Please let me know if any of these settings worked for you or if I need to further analyze your request.

Thank you!

**Reply by Tean** [reply]
Thanks, it worked
